xarray/core/dataset.py

```diff
@@ -64,6 +64,7 @@
     default_indexes,
     isel_variable_and_index,
     propagate_indexes,
+    remove_unused,
```

Comment:
minor comment: I think `remove_unused_levels_categories` is a nicer name because it's more descriptive